src/index.html

```diff
@@ -12,7 +12,57 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="stars__body">
+
+    <container class="stars__container">
+
+      <article class="stars stars--0">
+        <div class="stars_star"></div>
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. The class `stars_star` seems to represent the style or tag name. Consider renaming it to something that reflects its content or function, like `star` or `rating-star`.